Makefile

```diff
@@ -7,13 +7,16 @@ client/client: $(wildcard client/*.go) $(wildcard types/*.go)
 	cd client && go build
 
 example-target: check-env
-	AFL_HARDEN=1 $(AFL)/afl-clang -o example/server/$@ example/server/target.c
+	AFL_HARDEN=1 $(AFL)/afl-clang -o examples/server/$@ examples/server/target.c
 
-example-server: server/server
-	$(CURDIR)/server/server $(CURDIR)/example/server
+example-server-c: server/server
```

Comment:
These should both be marked `.PHONY` at the bottom since it doesn't produce a concrete artifact